invisible_cities/database/check_gain_update.py

```diff
@@ -0,0 +1,65 @@
+import sys
+import sqlite3
+import pymysql
+pymysql.install_as_MySQLdb()
+import numpy as np
+
+
+def connect_sqlite(dbname):
+    dbfile        = f'invisible_cities/database/localdb.{dbname}.sqlite3'
+    conn_sqlite   = sqlite3.connect(dbfile)
+    cursor_sqlite = conn_sqlite.cursor()
+    return cursor_sqlite
+
+
+def connect_mysql(dbname):
+    conn_mysql  = pymysql.connect(host="neutrinos1.ific.uv.es",
+                                  user='nextreader',passwd='readonly', db=dbname)
+    cursor_mysql  = conn_mysql .cursor()
+    return cursor_mysql
```

Comment:
Do these two functions not already exist somewhere else in our code?

Reply:
Not really. [Here](https://github.com/next-exp/IC/blob/e3383b08c72e7aa6b9f579a75b824b0ffa314d9e/invisible_cities/database/download.py#L61) two connections with their cursors are created but they are not in a separate function.

I will see if I can manage to refactor those lines and call them from the Github action.

Reply:
I have refactored those functions and tested again the code in my fork. The new version works well too.
One minor detail: in the files `invisible_cities/database/download.py` and `invisible_cities/database/check_gain_update.py` I have not used relative imports because we want to be able to run those scripts independently as:

 `python invisible_cities/database/download.py`
 `python invisible_cities/database/check_gain_update.py`

Reply:
It's probably a good idea to explain the inconsistent-with-the-rest-of-IC choice of absolute imports, in a comment in the code itself, lest someone wonder later on, and waste time on refactoring (and breaking) it.

Reply:
You are right, I have added comments about that to both files.